python/hail/docs/tutorials/01-genome-wide-association-study.ipynb

```diff
@@ -900,20 +831,11 @@
     "                      PC1=pca_scores.scores[0],\n",
     "                      PC2=pca_scores.scores[1])).to_pandas()\n",
     "\n",
-    "p = figure(title='PCA', background_fill_color='#EEEEEE',\n",
-    "          x_axis_label='PC1', y_axis_label='PC2')\n",
-    "pop_colors = {'AFR': 'green', 'EAS': 'black', 'EUR': 'blue', 'SAS': 'cyan', 'AMR': 'red'}\n",
+    "pops = pca_table[\"SuperPopulation\"].tolist()\n",
```

Comment:
Let's get rid of `to_pandas()` in pca_table.  I think pca_table can look something like:

```
pca_table = (
    pca_scores.select(SuperPopulation=common_ds.cols()[pca_scores.s].SuperPopulation,
                      PC1=pca_scores.scores[0],
                      PC2=pca_scores.scores[1])).collect()
```

Then `pca_scores` will be a `list` of `hl.Struct` that basically behaves like a Python `dict`.  Therefore, instead of `pca_table["PC1"].tolist()` you can write `[x.PC1 for x in pca_scores]`, etc.